lib/mixduty/incidents.ex

```diff
@@ -86,10 +86,34 @@ defmodule Mixduty.Incidents do
 
   """
   def list_business_services_impacted(client, incident_id) do
-      @path
-      |> Path.join(incident_id)
-      |> Path.join("business_services")
-      |> Path.join("impacts")
-      |> get(client)
+    @path
+    |> Path.join(incident_id)
+    |> Path.join("business_services")
+    |> Path.join("impacts")
+    |> get(client)
+  end
+
+  @doc """
+  Create an incident stakeholder status update
+  #### Examples
+      Mixdity.Incident.create_status_update(client, "This is status update", "P00PBUG")
+  """
+  @spec create_status_update(
+          client :: Mixduty.Client.t(),
+          status_update :: String.t(),
+          incident_id :: Stringt.t()
+        ) ::
+          %{required(:status_code) => integer(), required(:data) => map()}
+          | {:error, any()}
+          | {:error, String.t(), any()}
+  def create_status_update(client, status_update, incident_id) do
```

Comment:
I think `incident_id` should be in second place, just to be consistent with other methods?